src/show/pages/public/Explore/index.js

```diff
@@ -24,13 +25,24 @@ const Explore = () => {
 
   useEffect(() => {
     axios
-      .get(`http://localhost:3000/v1/projects?category=${selectedCategory}`)
+      .get(
+        `http://localhost:3000/v1/projects?category=${selectedCategory}&search=${search}`
+      )
       .then(res => setProjects(res.data))
       .catch(err => err);
-  }, [selectedCategory]);
+  }, [selectedCategory, search]);
+
+  // className='flex justify-end items-center border border-white hover:border-primary-100 hover:px-3 w-64 h-8 rounded-full mr-5'
```

Comment:
No commented code like this should get into the main code. Delete.

Reply:
Ooops